src/components/apps/launch/InputSelector.js

```diff
@@ -97,17 +97,12 @@ const InputSelector = ({
     const { t } = useTranslation("launch");
     const { t: prefI18n } = useTranslation("preferences");
     const { setFieldValue } = form;
-    const [bootstrapInfo, setBootstrapInfo] = useBootstrapInfo();
+    const bootstrapInfo = useBootstrapInfo()[0];
 
     //update last folder used.
-    const [mutatePreferences] = useSavePreferences(
-        ({ preferences }) => {
-            setBootstrapInfo({ ...bootstrapInfo, preferences });
-        },
```

Comment:
No need to update bootstrap context here. That already happens in `useSavePreferences`